### PR TITLE
Fix flow run activity chart

### PR DESCRIPTION
**Pull request by pleek91**
# Description
Fixes an issue where when viewing future times could result in bars overflowing the activity.

### CODE REVIEW

File: src/components/FlowRunsBarChart.vue [hunk 1]
```diff
@@ -57,6 +57,10 @@
       return null
     }
 
+    if (barsDebounced.value === 0) {
+      return null
+    }
```
Comment (pleek91) [inline src/components/FlowRunsBarChart.vue:62]: Unrelated fix. Noticed that before the chart was sized it would sometimes make a request with a limit of `0`. Which of course would return 0 flow runs. So we don't need to make a request for that.

**Review by znicholasbrown — APPROVED** [review]
